### ISSUE TITLE
This extension violates the Chrome Web Store policy.

### ISSUE THREAD

**Issue by d73** [pmsosa/Hidden#4]
Extension recently (started 5/23 afaik) started getting automatically disabled. The error Chrome gives is:
This extension violates the Chrome Web Store policy.

Extension version 1.3
Chrome version 90.0.4430.212 (Official Build) (64-bit)

**Comment by stk21** [comment]
same issue here![image](https://user-images.githubusercontent.com/32824148/119841137-cd2ad000-bec2-11eb-9c80-2a48f2a17688.png)

**Comment by div4d** [comment]
yes, same problem here. Chrome is now disabling the extension every restart. is there a way to fix this? thanks

**Comment by kwmkjf** [comment]
Same Problem for me, would love to use this extension again!

**Comment by itxch** [comment]
Hey, @d73 @stk21 @div4d @kwmkjf my solution was to clone the repo and load the extension myself into google chrome. This just requires enabling dev mode in Chrome extension page and loading in an unpacked extension.

**Comment by UNIVAC-Colonel-Panic** [comment]
I'm definitely interested. I've been using [Incognito Regex \(Fixed Version\) \- Chrome Web Store](https://chrome.google.com/webstore/detail/incognito-regex-fixed-ver/okaigpjdjpcecnghfiegfopahbkcejak) but it doesn't work consistently.

**Comment by itxch** [comment]
> I'm definitely interested. I've been using [Incognito Regex \(Fixed Version\) \- Chrome Web Store](https://chrome.google.com/webstore/detail/incognito-regex-fixed-ver/okaigpjdjpcecnghfiegfopahbkcejak) but it doesn't work consistently.

Let me know, how it goes for you if steps above don't work.
